core/components/header/schema.ts

```diff
@@ -0,0 +1,12 @@
+import { z } from 'zod';
+
+import type { CurrencyCode } from './fragment';
+
+// eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+export const currencyCodeSchema = z
```

Comment:
:nit: CapitalCase for zod schemas
```suggestion
export const CurrencyCodeSchema = z
```